libs/service-portal/signature-collection/src/lib/messages.ts

```diff
@@ -245,4 +246,110 @@ export const m = defineMessages({
     defaultMessage: 'Heimilisfang',
     description: '',
   },
+
+  /* Parliamentary */
+  parliamentaryElectionsTitle: {
+    id: 'sp.parliamentaryElectionsTitle',
```

Comment:
Namespaces are missing 

Should be like this: 
    **id: 'sp.signatureCollection:title',**

Reply:
Still missing namespaces.

instead of: 
id: 'sp.parliamentaryElectionsTitle'

it should be like this: 
id: sp.parliamentary:elections-title

where the "sp.parliamentary" is the same for everything related then the different strings should come after the **:**

Reply:
oh dam 😆

Reply:
hmm.. seems ive updated them all to be of this format:` id: 'sp.signatureCollection:xxx',`

Reply:
I was opening the outdated version 😅 ✅